Automattic/jetpack
# Add the template for the response view in the new feedback dashboard

## Proposed changes:


This PR implements the response view template for the new feedback dashboard. It's just the static template, and it'll be hooked up with the feedback dashboard state in a followup PR (we're still working on figuring out how the backend and thus the state will look).

<img width="508" alt="Screenshot 2023-02-09 at 19 00 11" src="https://user-images.githubusercontent.com/8056203/217900340-9f6befa9-6130-4379-b0bf-a62e83bd941f.png">

### Other information:


## Jetpack product discussion



pcsBup-Pl-p2

## Does this pull request change what data or activity we track or use?



No.

## Testing instructions:






Enable the new feedback dashboard with these filters:

```
add_filter( 'jetpack_contact_form_use_package', '__return_true' );
add_filter( 'jetpack_forms_dashboard_enable', '__return_true' );
```

- Go to `/wp-admin/admin.php?page=jetpack-forms`.
- You should see the new template in the response part of the layout.
